crates/tessellation/src/stroke.rs

```diff
@@ -1680,7 +1680,9 @@ fn compute_join_side_positions_fixed_width(
     //    flattened segment.
     let extruded_normal = front_normal * vertex.half_width;
     let unclipped_miter = (join.line_join == LineJoin::Miter || join.line_join == LineJoin::MiterClip)
-        && !miter_limit_is_exceeded(extruded_normal, miter_limit);
+        && !miter_limit_is_exceeded(front_normal, miter_limit);
+
+    println!("join {:?} prev tan {prev_tangent:?} next tan {next_tangent:?} normal {normal:?} miter exceeded: {:?}", join.position, miter_limit_is_exceeded(front_normal, miter_limit));
```

Comment:
to be sure, did you mean to leave the println in? :sweat_smile:

Reply:
oh snap.

Reply:
Fixed in 1.0.10. Thanks for pointing it out!